society-of-biblical-literature-fullnote-bibliography.csl

```diff
@@ -877,6 +917,12 @@
         <group prefix=" (" suffix=")">
           <group delimiter=", ">
             <group delimiter=" ">
+          <text macro="event-note"/>
+          <group delimiter="; ">
+            <text macro="originally-published"/>
+            <group delimiter=", ">
+              <text macro="reprint-note"/>
+              <text macro="publisher"/>
```

Comment:
Yeah, so by moving the publisher down here into the group, you get the year preceded by a semicolon rather than a comma for items without original date, which is incorrect. That's just for notes, it's fine for the bibliography

Reply:
I have reworked the commit to fix the issue.